TestsOutsidePackage/test_score.py

```diff
@@ -2,22 +2,61 @@
 #  Copyright (c) Microsoft Corporation. All rights reserved.
 #  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 #  ------------------------------------------------------------------------------------------
+from pathlib import Path
+from typing import List
+from unittest import mock
+import zipfile
 import numpy as np
 import pytest
 from pytorch_lightning import seed_everything
 
+from InnerEye.Azure.azure_config import AzureConfig
 from InnerEye.Common.fixed_paths_for_tests import full_ml_test_data_path
 from InnerEye.Common.output_directories import OutputFolderForTests
+from InnerEye.ML.run_ml import MLRunner
 from InnerEye.ML.utils import io_util
 from InnerEye.ML.utils.io_util import reverse_tuple_float3
 from Tests.ML.configs.DummyModel import DummyModel
 from Tests.ML.utils.test_model_util import create_model_and_store_checkpoint
-from score import create_inference_pipeline, is_spacing_valid, run_inference
+from passthrough_model import PassThroughModel
+from score import create_inference_pipeline, is_spacing_valid, run_inference, score_image, ScorePipelineConfig, \
+    extract_zipped_dicom_series, convert_zipped_dicom_to_nifti, \
+    convert_nifti_to_zipped_dicom_rt
+
 
 test_image = full_ml_test_data_path("train_and_test_data") / "id1_channel1.nii.gz"
 img_nii_path = full_ml_test_data_path("test_img.nii.gz")
 
 
+# The directory containing this file.
+THIS_DIR: Path = Path(__file__).parent.resolve()
+# The TestData directory.
+TEST_DATA_DIR: Path = THIS_DIR / "TestData"
+# Filenames of dcm files in flat test zip.
+TEST_FLAT_ZIP_FILENAMES = ['1.txt', '2.dcm', '3.dcm', '4.dcm', '5.txt']
+# Flat test zip file.
+TEST_FLAT_ZIP_FILE: Path = TEST_DATA_DIR / "test_flat.zip"
+# As test_flat but everything in "folder1"
+TEST_FLAT_NESTED_ZIP_FILE: Path = TEST_DATA_DIR / "test_flat_nested.zip"
+# As test_flat_nested but everything in "folder2"
```

Comment:
While we are at the point of space saving: Can we replace all ZIP files with something created on-the-fly? At the moment, only you know what the contents of those ZIP files is, and why they need to be what they are.